pkg/instances/instances.go

```diff
@@ -30,7 +30,7 @@ const (
 )
 
 var (
-	controlPlaneLabels = [...]string{"dapr-operator", "dapr-sentry", "dapr-placement", "dapr-sidecar-injector"}
+	controlPlaneLabels = [...]string{"dapr-operator", "dapr-sentry", "dapr-placement-server", "dapr-sidecar-injector", "dapr-dashboard"}
```

Comment:
add back `dapr-placement` until 1.0.0 is released and then it can be removed in a future release.

Reply:
We don't use old version of placement anymore. But, I will add it.